source/binding/Statements.cpp

```diff
@@ -995,6 +1033,32 @@ bool CaseStatement::verifyConstantImpl(EvalContext& context) const {
     return true;
 }
 
+void CaseStatement::serializeTo(ASTSerializer& serializer) const {
+    serializer.write("condition", toString(condition));
+    serializer.write("check", toString(check));
+    serializer.startArray("items");
+    for (auto const& item : items) {
+        serializer.startObject("item");
```

Comment:
This is still not quite correct. If you run this code you'll see that it produces JSON like items=[item={}], but that's not valid JSON. Array elements shouldn't have property names. Instead for each item you just want to start an object, with no property name.

Reply:
What I had in mind is something like this:
```JSON
"items": [
    {
        "expressions": [],
        "stmt": stmt
    },
    {
        "expressions": [],
        "stmt": stmt
    }
]
```

I will double check with the output.

Reply:
Yeah, in that case you need a startObject() call that doesn't write the property name.

Reply:
Thanks. I've validated the JSON output from case statement and it seems to be correct.